deployment/docker-compose/docker-compose.yml

```diff
@@ -18,7 +18,18 @@ services:
       watch:
         - action: rebuild
           path: ../../backend
-
-
+  chainlit-app:
+    build:
+      context: ../../chainlit
+      dockerfile: Dockerfile.chainlit
+    ports:
+      - "8100:8100"
+    volumes:
+      - .:/app  # Mount your project folder
```

Comment:
Suggest removing comment. Best practice to avoid making the codebase cluttered is to only add comments when the code is not obvious or the decision is no obvious.